test/tasks_test.exs

```diff
@@ -1,8 +1,27 @@
 defmodule TasksTest do
   use ExUnit.Case
-  doctest Tasks
 
-  test "greets the world" do
-    assert Tasks.hello() == :world
+  import Tasks
```

Comment:
Just a comment: 

We use `import` only with common helper function. Something that is called all over the project. Like
`create_admin_user`.

So we would use `Tasks.fib(5)`

An example from real world is a call to

`Banmed.Account.create(params)`

This tells you where the code is (importing `Banmed.Account` would hide it) and what the code does. Also, we don't need to name our fn `create_account`, it's in the Account module so you know what it creates :)

This doesn't have much to do with your code and test.